pkgs/tools/package-management/nix/default.nix

```diff
@@ -233,23 +207,6 @@ lib.makeExtensible (
 
       latest = self.nix_2_29;
 
-      # The minimum Nix version supported by Nixpkgs
-      # Note that some functionality *might* have been backported into this Nix version,
-      # making this package an inaccurate representation of what features are available
-      # in the actual lowest minver.nix *patch* version.
-      minimum =
-        let
-          minver = import ../../../../lib/minver.nix;
-          major = lib.versions.major minver;
-          minor = lib.versions.minor minver;
-          attribute = "nix_${major}_${minor}";
-          nix = self.${attribute};
-        in
-        if !self ? ${attribute} then
-          throw "The minimum supported Nix version is ${minver} (declared in lib/minver.nix), but pkgs.nixVersions.${attribute} does not exist."
-        else
-          nix;
-
```

Comment:
Should this alias to 2.24 as the oldest supported (and packaged) version?

It feels slightly out-of-place for us to remove the `nixVersion.miniumum` attribute entirely, without input from the nix team.

That said, if the nix team are ok with removing the attribute, then I'm fine with it.